rdt-app/src/main/java/io/ona/rdt_app/fragment/RDTJsonFormFragment.java

```diff
@@ -30,8 +40,18 @@ public static JsonFormFragment getFormFragment(String stepName) {
     }
 
     @Override
-    protected void initializeBottomNavigation(JSONObject step, View rootView) {
+    protected void initializeBottomNavigation(final JSONObject step, final View rootView) {
         super.initializeBottomNavigation(step, rootView);
+        // Handle initialization of the countdown timer bottom navigation
+        String currStep = "step" + currentStep;
+        boolean buttonEnabled = true;
+        if ("step7".equals(currStep)) {
+            buttonEnabled = false;
+            setNextButtonState(rootView, buttonEnabled);
+        } else if ("step8".equals(currStep)) {
```

Comment:
This should just be else, and not specific to step8

We mentioned the case of possibly going to a previous?

Reply:
Yeah. The button should be enabled by default so the logic should do that and disable only on that step :+1: